workflow-bot-app/src/test/java/com/symphony/bdk/workflow/SendMessageIntegrationTest.java

```diff
@@ -142,6 +152,30 @@ void sendMessageWithUidsVariables() throws Exception {
     assertThat(workflow).isExecuted().hasOutput(String.format(OUTPUTS_MSG_KEY, "sendMessageWithUserIds"), message);
   }
 
+  @Test
+  void sendMessageWithTemplateSuccessfull() throws IOException, ProcessingException {
+    final Workflow workflow =
+        SwadlParser.fromYaml(getClass().getResourceAsStream("/message/send-message-with-freemarker.swadl.yaml"));
+
+    TemplateEngine templateEngine = mock(TemplateEngine.class);
+    Template template = mock(Template.class);
+    when(messageService.templates()).thenReturn(templateEngine);
+    when(templateEngine.newTemplateFromFile(any())).thenReturn(template);
+    when(template.process(anyMap())).thenReturn("Hello world!");
+
+    engine.deploy(workflow);
+    engine.onEvent(messageReceived("/send"));
+
+    assertThat(workflow).isExecuted();
+    ArgumentCaptor<Message> messageArgumentCaptor = ArgumentCaptor.forClass(Message.class);
+    ArgumentCaptor<Map<String, Object>> paramsArgumentCaptor = ArgumentCaptor.forClass(Map.class);
+
+    verify(template, times(1)).process(paramsArgumentCaptor.capture());
+    assertThat(paramsArgumentCaptor.getValue()).contains(Map.entry("variables",  Map.of("val","world")));
+    verify(messageService, times(1)).send(eq("123"), messageArgumentCaptor.capture());
+    AssertionsForClassTypes.assertThat(messageArgumentCaptor.getValue().getContent()).isEqualTo("<messageML>Hello world!</messageML>");
```

Comment:
is that the proper assertThat to import? (vs the one above)

Reply:
removed the import in the next commit